numba/core/codegen.py

```diff
@@ -829,6 +829,8 @@ def _customize_tm_options(self, options):
         arch = ll.Target.from_default_triple().name
         if arch.startswith('x86'): # one of x86 or x86_64
             reloc_model = 'static'
+        elif arch.startswith('ppc'):
+            reloc_model = 'pic'
```

Comment:
Perhaps add a note here about how this is entirely different to that in LLVM's `getEffectiveRelocModel` etc itself, but seems to work. I had another vague recollection about this yesterday regarding PIC leading to giant amounts of mapped memory on PPC for LLVM 7/8, worth taking a look?

Reply:
I ran the full testsuite on PPC and it passes basically everything. Is that a good sign?

Reply:
Yes!